ydb/core/statistics/aggregator/tx_aggr_stat_response.cpp

```diff
@@ -0,0 +1,122 @@
+#include "aggregator_impl.h"
+
+#include <ydb/core/statistics/stat_service.h>
+
+namespace NKikimr::NStat {
+
+struct TStatisticsAggregator::TTxAggregateStatisticsResponse : public TTxBase {
+    NKikimrStat::TEvAggregateStatisticsResponse Record;
+
+    bool SendReqDistribution = false;
+    bool SendAggregate = false;
```

Comment:
NIT:
If I'm right only one of this bool can be true at the same time.
Maybe it will be more clear to make this enum
```cpp
enum class EAction : ui8 {
   None,
   SendReqDistribution,
   SendAggregate,
};
```